.github/labelChecker/index.js

```diff
@@ -0,0 +1,34 @@
+const rm = require('typed-rest-client/RestClient');
+const core = require('@actions/core');
+const github = require('@actions/github');
+
+async function main() {
+    try {
+        const pullRequestNumber = github.context.issue.number;
+        console.log(`Running for PR: ${pullRequestNumber}\n`);
+        let rest = new rm.RestClient('labelChecker');
+        console.log('Getting label info\n');
+        let res = await rest.get(`https://api.github.com/repos/microsoft/azure-pipelines-agent/issues/${pullRequestNumber}/labels`);
+        console.log(`Labels: ${JSON.stringify(res.result)}`);
+        let labelCount = 0;
+        res.result.forEach(tag => {
+            let name = tag.name.toLowerCase();
+            if (['bug', 'enhancement', 'misc', 'internal'].indexOf(name) > -1) {
+                console.log(`Found tag: ${name}`);
+                labelCount++;
+            }
+        });
+
+        if (labelCount === 0) {
+            throw 'Must be labeled one of [bug, enhancement, misc, internal]'
```

Comment:
can we make the list ['bug', 'enhancement', 'misc', 'internal'] a constant array and use that instead of having in separate strings and in a forEach()?

Reply:
Done!